libs/view-transition/src/services/view-transition.service.ts

```diff
@@ -6,10 +6,12 @@ import {Observable, throwError} from 'rxjs';
     providedIn: 'root',
 })
 export class ViewTransitionService {
-    private readonly isSupported = 'startViewTransition' in this.document;
-
     constructor(@Inject(DOCUMENT) private readonly document: Document) {}
 
+    get isSupported(): boolean {
+        return 'startViewTransition' in this.document;
```

Comment:
Why it should be getter ?

Reply:
<img width="819" alt="image" src="https://github.com/taiga-family/ng-web-apis/assets/12021443/f3fb7804-6cea-4091-b864-68c848a4da28">